mpas_analysis/obs/observational_datasets.xml

```diff
@@ -531,7 +614,7 @@
       - indexNino34
     </tasks>
     <subdirectory>
-      Ocean/Nino/ERS_SSTv4
```

Comment:
Same as above, the subdirectory is necessary to keep from overwriting the README for the other observations.